pkg/comp-functions/functions/vshnnextcloud/deploy.go

```diff
@@ -136,72 +131,6 @@ func DeployNextcloud(ctx context.Context, svc *runtime.ServiceRuntime) *xfnproto
 	return nil
 }
 
-func addPostgreSQL(svc *runtime.ServiceRuntime, comp *vshnv1.VSHNNextcloud) error {
```

Comment:
Why are we removing this?

Reply:
it's dead code. vshnnextcloud uses `common.NewPostgreSQLDependencyBuilder()` that you implemented according to blame :D

Reply:
perfect then you fixed my bug, thank you :D